osu.Game/Online/API/APIAccess.cs

```diff
@@ -165,7 +165,7 @@ private void run()
                         }
 
                         // The Success callback event is fired on the main thread, so we should wait for that to run before proceeding.
-                        // Without this, we will end up circulating this Connecting loop multiple times and queueing up many web requests
+                        // Without this, we will end up circulating this Connecting loop multiple times and queuing up many web requests
```

Comment:
This is just picking at personal preference with words that have more than one accepted spelling.  Unnecessary.